signup/models.py

```diff
@@ -285,6 +287,15 @@ def verification_key_expired(self):
                (start_at + expiration_date <= datetime_or_now())
     verification_key_expired.boolean = True
 
+    @property
+    def picture_url(self):
```

Comment:
I don't understand the need for this method. Is the full URL not stored in the `picture` field?